proteus/config/jim.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import
+from .default import *
+    
+#platform_extra_compile_args = ['-DPETSC_INCLUDE_AS_C']
+    
+PROTEUS_PRELOAD_LIBS=['/p/home/apps/hpe/mpt-2.21/lib/libmpi.so','/opt/hpe/hpc/mpt/mpt-2.21/lib/libmpi_lustre.so']
+#PROTEUS_PRELOAD_LIBS=['/p/home/apps/sgi/mpt-2.12-sgi712r26/lib/libmpi.so',
```

Comment:
Are the commented-out lines going to be useful?

Reply:
Only as a record to help somebody in the future trying to find the kinds of places where they've put these libraries. It's in the history, though, so I'll remove it.